collector/redfish_collector.go

```diff
@@ -0,0 +1,244 @@
+package collector
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"hardware_exporter/lib/gofish"
```

Comment:
base on `github.com/kubeservice-stack/hardware-exporter` relative path

Reply:
This gofish package I have change a lot. So I download it.